src/snowflake/snowpark/mock/_functions.py

```diff
@@ -495,45 +501,63 @@ def mock_to_time(
             if data is None:
```

Comment:
NIT: can we rewrite this to use `ColumnEmulator.apply` instead of a for loop?

Reply:
it probably takes some extra time to rewrite so I would keep it for now